File: python/7 kyu/328.py
# https://www.codewars.com/kata/5a0d38c9697598b67a000041/train/python

# Passed

def eliminate_unset_bits(number):
    result_in_binary = number.replace('0', str())
    result = int(result_in_binary if result_in_binary else '0', 2)
    return result

output = eliminate_unset_bits("11010101010101")
print(output)

File: nodejs/8 kyu/328.js
// https://www.codewars.com/kata/5875b200d520904a04000003/train/javascript

// Passed

function enough(cap, on, wait) {
  let totalPeople = on + wait;
  let result;
  if (totalPeople <= cap) {
    result = 0;
  } else {
    result = totalPeople - cap;
  }
  return result;
}

const output = enough(100, 60, 50);
console.log(output);